### ISSUE TITLE
ARC A380 Fallout76 really poor peerfomance

### ISSUE THREAD

**Issue by J0dla** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#400]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Game is not cracked, modded and use the latest patch

### Game / Application [Required]

Fallout76

### Game Platform [Required]

- [X] Steam
- [ ] Microsoft Store
- [ ] GOG Galaxy
- [ ] Ubisoft Connect
- [ ] Battle.net
- [ ] Epic Games
- [ ] Rockstar Games Launcher
- [ ] EA app
- [ ] Other (please specify)

### Processor / Processor Number [Required]

5700G

### Graphic Card [Required]

ARC A380

### GPU Driver Version [Required]

- [X] 31.0.101.4575
- [X] 31.0.101.4514
- [ ] 31.0.101.4502
- [ ] 31.0.101.4499
- [ ] 31.0.101.4382
- [ ] 31.0.101.4369
- [ ] 31.0.101.4335
- [ ] 31.0.101.4314
- [ ] 31.0.101.4311
- [ ] 31.0.101.4257
- [ ] 31.0.101.4255
- [ ] 31.0.101.4148
- [ ] 31.0.101.4146
- [ ] 31.0.101.4125
- [ ] 31.0.101.4123
- [ ] 31.0.101.4091
- [ ] 31.0.101.4090
- [ ] 31.0.101.4034

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [X] DirectX11
- [ ] DirectX10
- [ ] DirectX9

### Windows Build Number [Required]

- [X] Windows 11 22H2
- [ ] Windows 11 21H2
- [ ] Windows 10 22H2
- [ ] Windows 10 21H2
- [ ] Other (Please specify)

### Intel System Support Utility report

[igcit_ssu.zip](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/files/12088795/igcit_ssu.zip)

### Description and steps to reproduce [Required]

General bad game experience, no matter what ingame settings poor frametime.
It gets <20fps on events like Test Your Metal, Swarm of Suitors.
Small example video clip :
https://youtu.be/fTegvWSRawM

### Game graphic quality [Required]

- [X] Low
- [X] Medium
- [X] High
- [ ] Ultra

### Game resolution [Required]

1920x1080

### Game VSync [Required]

On

### Game display mode [Required]

- [ ] Fullscreen
- [ ] Windowed
- [X] Borderless Windowed

### Detailed game settings [Required]

![CONFIG_1](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/17433864/9c6844c1-8c1f-4e25-a84e-50be1bda4e7e)
![CONFIG_2](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/17433864/573c4e8b-5397-4985-8cdd-8e849cb6b1ab)
![CONFIG_3](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/assets/17433864/b2b2576d-bac9-4a72-a5c5-a93a3a548a3b)

**Comment by Karen-Intel** [comment]
Hey @J0dla currently debugging this game
Question: have you tested w/driver v.4576? I'm still in the early stages of the game and I have an average of 64 FPS
I wonder if in these events the performance is still low with this new driver, can you check?

Thanks

Karen

**Comment by J0dla** [comment]
I updated the driver. Little warning this games uses mandatory VSYNC.
I suppose your tests were performed on starting area (Forest region) and probably with A770 on A380 my avg  there is about 40fps and 1% about 22 but take in mind this region is one of the "empties". I had redo train depot test with same sad results. The problem is if you play with others or go to public events (unfortunately they are random time map events so reproducing would take time) game is stutter mess. I do some tests with my friend that have 1660 and the difference is night and day. I know this game uses older api but unfortunately i cant count on Bethesda to update their modified creation engine. The lows are the biggest problem. I will do more testing but yeah the performance that arc shows here is really bad...

**Comment by akarnokd** [comment]
Sorry to chime in like this.

I run an A770 LE with driver 4502 on a 13700K. Can't test newer drivers due to some reporting the return of the audio stutter issues fixed with 4369 or so.

I play Fallout 76 daily and I've been getting wonky framerates, even without much happening at all. The best places to check out is *Watoga EMS upstairs*, *Uncanny caverns* & *West Tek*. The card barely draws 100 W and is 60-70% utilized @ 1080p 60Hz VSync On, windowed fullscreen, high textures, low everything else. If I disable VSync, I get framerate fluctuation between 30 & 130 FPS, quite noticeably. If you want test with lots of other players and lots of effects, visit *Test Your Metal* or *Moonshine Jamboree* public events when they are on.

DXVK is not an option. I haven't tried it because there were reports getting banned for it before. Now, the game won't even launch with it.

**Comment by Karen-Intel** [comment]
Ty all for sharing all these details. 
We're already making sure to test this on our end and like you mentioned, it might take some time. 
However, once we're able to repro and gather the required logs, will be reporting this internally so the development team is aware of this. 
Something to mention is that we do not have an A380 atm, but I'm sure we will get good evidences on ARC as well.
If you have other friends or acquaintances that are facing the same performance issues, they can use this same thread :)

Karen

**Comment by LadyMikea** [comment]
Well, If I can chime in with some what similar report. I can confirm abysmal performance but with Arc 750 and Fallout 4 (but these games shares the same engine(?), driver 4577. It has a typical behavior of "bad" DX11 game, while looking at the ground or sky, you'll get 60+ FPS, but looking straight, it is about 10 or less. Using DXVK (mind you, I'm talking about regular Fallout 4 not the online one) gets you a 60+ FPS with no problem.

**Comment by fuchaohan** [comment]
U can try  DXVK , can add about 30 frames.

**Comment by LadyMikea** [comment]
Yeah, with help of DXVK Fallout 4 (as well as 3 and NV) are fine (very fine in fact), but my point was about poor  DX 11 "driver" implementation. Driver 4644 (the one with "better" DX11,  supposedly) shows no sign of better performance in such DX 11 titles as -  Fallout 4, Elex, Elex II - they're slow as measles (unless you are looking at ground...), or as in case of War World Z, shows flashing  textures and glitches. Maybe future drivers will provide more universal DX 11 solution.

**Comment by akarnokd** [comment]
I'm experiencing improved performance with 4953 on an A770LE.

Places such as West Tek interior, Palace of the Winding Path, Burning Mine runs 55-60 now (used to dip to 20). Using the flamer around mobs no longer tanks framerate either. Haven't tried big events yet.

**Comment by SavageWaffle** [comment]
This would appear resolved, lets close the ticket
